### DISCUSSION TITLE
How to compile sdkjs/common/allFonts.js

### DISCUSSION THREAD

**Discussion by fenghw2087** [ONLYOFFICE/build_tools#420] [Q&A] [answered]
I want to run the development environment.
1. grunt develop in project sdkjs
2. grunt develop in project web-apps
3. put those two projects in a nginx server
4. in my own project, I import "api.js" like this
`<script
            type="text/javascript"
            src="http://mylocalip/web-apps/apps/api/documents/api.js"
        ></script>`
        
then I met a error:
![image](https://user-images.githubusercontent.com/40625210/148315595-1957d75d-4cfd-40ba-b8f3-02f721515a93.png)
It seems I forgot to compile allFonts.js

Does anyone knows how to compile it?

**Comment by ShockwaveNN** [comment, marked as answer]
`AllFonts.js` is generated by this command

https://github.com/ONLYOFFICE/build_tools/#generate-fonts-data

And moving this issue to `build_tools` repo, since all problems related to build is discussed there

**Reply by fenghw2087** [reply]
> `AllFonts.js` is generated by this command
> 
> https://github.com/ONLYOFFICE/build_tools/#generate-fonts-data
> 
> And moving this issue to `build_tools` repo, since all problems related to build is discussed there

thanks
